docker-compose.yml

```diff
@@ -0,0 +1,57 @@
+---
```

Comment:
why is there a docker-compose.yml file added?

Reply:
> I don't have a compatible local setup

> Edit: added a docker-compose.yml file for easy running of specs on local.

We can worry about it later.

Reply:
I can drop the commit if you don't need it and just keep that as a snippet for myself for future contributions.

Reply:
If you could drop it, that'd be great (otherwise we'd come back to it when ready to merge). It's not something we generally use and maintaining it becomes difficult in that case.

Reply:
done